test/e2e/deploy/deploy.go

```diff
@@ -1051,30 +1051,48 @@ func EnsureDeploymentNow(f *framework.Framework, deployment *Deployment) {
 			running.Name(), running.Namespace,
 			deployment.Name(), deployment.Namespace)
 
-		if running.HasOperator {
-			cmd := exec.Command("test/stop-operator.sh")
-			if running.HasOLM {
-				cmd.Args = append(cmd.Args, "-olm")
-			}
-			// Keep both the CRD and namespace in case the required deployment is also the operator
-			// required for version skew tests, otherwise it brings down the existing driver deployment(s)
-			if deployment.HasOperator {
-				cmd.Args = append(cmd.Args, "-keep-crd", "-keep-namespace")
+		// If both running and wanted operator deployments are managed by
```

Comment:
The commit message has a typo: s/vesion-skew/version-skew/

Reply:
Fxied.